src/include/OSL/llvm_util.h

```diff
@@ -665,11 +665,17 @@ class OSLEXECPUBLIC LLVM_Util {
     /// Return an llvm::Constant holding the given floating point constant.
     llvm::Constant* constant(float f);
 
+    /// Return an llvm::Constant holding the given floating point constant.
+    llvm::Constant* constant64(double f);
```

Comment:
overloading constant64 creates some ambiguity as seen in the CI:
```
19: error: call to member function 'constant64' is ambiguous
        return ll.constant64(str.hash());
               ~~~^~~~~~~~~~/Users/runner/work/OpenShadingLanguage/OpenShadingLanguage/src/include/OSL/llvm_util.h:669:21: note: candidate function
    llvm::Constant* constant64(double f);
                    ^
/Users/runner/work/OpenShadingLanguage/OpenShadingLanguage/src/include/OSL/llvm_util.h:680:21: note: candidate function
    llvm::Constant* constant64(uint64_t i);
                    ^
```
So for completeness probably need to add constant64(int64_t) to remove the ambguity.

Reply:
Yeah, the strange thing is `ustring::hash()` returns `uint64_t` so there should be no ambiguity.

There is already a `constanti64(int64_t)` which maybe should be renamed for consistency.

Reply:
Hmm, found this report:
https://github.com/dmlc/HalideIR/issues/17
https://en.cppreference.com/w/cpp/types/size_t
`typedef /*implementation-defined*/ size_t;`
So apparently we can't rely on size_t being uint64_t even on 64bit platform.

Since there error is coming from
```
/Users/runner/work/OpenShadingLanguage/OpenShadingLanguage/src/liboslexec/backendllvm.h:155:19: error: call to member function 'constant64' is ambiguous
        return ll.constant64(str.hash());
```
Could you try adding a conversion cast to that call:
```
    llvm::Value* llvm_load_stringhash(ustring str)
    {
        return ll.constant64(uint64_t(str.hash()));
    }
```
to see if we can make that happy.
Alternatively it appears ustring::hash now returns an actual uint64_t vs. size_t.
https://github.com/OpenImageIO/oiio/commit/7f28d3f3f16abce93b2f81180d7610ab5c5d078f
That is in master since v25.0.0-dev.  So could request the minimum OIIO version be moved up, but probably the cast is OK workaround for now.